processing/src/main/java/io/druid/query/search/search/DefaultSearchQueryMetrics.java

```diff
@@ -0,0 +1,238 @@
+  }
+
+  @Override
+  public void server(String host)
+  {
+    queryMetrics.server(host);
+  }
+
+  @Override
+  public void remoteAddress(String remoteAddress)
+  {
+    queryMetrics.remoteAddress(remoteAddress);
+  }
+
+  @Override
+  public void status(String status)
+  {
+    queryMetrics.status(status);
+  }
+
+  @Override
+  public void success(boolean success)
+  {
+    queryMetrics.success(success);
+  }
+
+  @Override
+  public void segment(String segmentIdentifier)
+  {
+    queryMetrics.segment(segmentIdentifier);
+  }
+
+  @Override
+  public void chunkInterval(Interval interval)
+  {
+    queryMetrics.chunkInterval(interval);
+  }
+
+  @Override
+  public void preFilters(List<Filter> preFilters)
+  {
+    queryMetrics.preFilters(preFilters);
+  }
+
+  @Override
+  public void postFilters(List<Filter> postFilters)
+  {
+    queryMetrics.postFilters(postFilters);
+  }
+
+  @Override
+  public BitmapResultFactory<?> makeBitmapResultFactory(BitmapFactory factory)
+  {
+    return queryMetrics.makeBitmapResultFactory(factory);
+  }
+
+  @Override
+  public QueryMetrics reportQueryTime(long timeNs)
+  {
+    return queryMetrics.reportQueryTime(timeNs);
+  }
+
+  @Override
+  public QueryMetrics reportQueryBytes(long byteCount)
+  {
+    return queryMetrics.reportQueryBytes(byteCount);
+  }
+
+  @Override
+  public QueryMetrics reportWaitTime(long timeNs)
+  {
+    return queryMetrics.reportWaitTime(timeNs);
+  }
+
+  @Override
+  public QueryMetrics reportSegmentTime(long timeNs)
+  {
+    return queryMetrics.reportSegmentTime(timeNs);
+  }
+
+  @Override
+  public QueryMetrics reportSegmentAndCacheTime(long timeNs)
+  {
+    return queryMetrics.reportSegmentAndCacheTime(timeNs);
+  }
+
+  @Override
+  public QueryMetrics reportIntervalChunkTime(long timeNs)
+  {
+    return queryMetrics.reportIntervalChunkTime(timeNs);
+  }
+
+  @Override
+  public QueryMetrics reportCpuTime(long timeNs)
+  {
+    return queryMetrics.reportCpuTime(timeNs);
+  }
+
+  @Override
+  public QueryMetrics reportNodeTimeToFirstByte(long timeNs)
+  {
+    return queryMetrics.reportNodeTimeToFirstByte(timeNs);
+  }
+
+  @Override
+  public QueryMetrics reportNodeTime(long timeNs)
+  {
+    return queryMetrics.reportNodeTime(timeNs);
+  }
+
+  @Override
+  public QueryMetrics reportNodeBytes(long byteCount)
+  {
+    return queryMetrics.reportNodeBytes(byteCount);
+  }
+
+  @Override
+  public QueryMetrics reportBitmapConstructionTime(long timeNs)
+  {
+    return queryMetrics.reportBitmapConstructionTime(timeNs);
+  }
+
+  @Override
+  public QueryMetrics reportSegmentRows(long numRows)
+  {
+    return queryMetrics.reportSegmentRows(numRows);
+  }
+
+  @Override
+  public QueryMetrics reportPreFilteredRows(long numRows)
+  {
+    return queryMetrics.reportPreFilteredRows(numRows);
+  }
+
+  @Override
+  public void emit(ServiceEmitter emitter)
+  {
+    queryMetrics.emit(emitter);
+  }
+
+  @Override
+  public void granularity(SearchQuery query)
```

Comment:
Please place this method along with other methods of it's group, i. e. after `queryId()`.